CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ option(ENABLE_UNDEFINED_SANITIZER "Use UB sanitizer for Debug build" OFF)
 option(ENABLE_TESTS "Enable Valhalla tests" ON)
 option(ENABLE_WERROR "Convert compiler warnings to errors. Requires ENABLE_COMPILER_WARNINGS=ON to take effect" OFF)
 option(ENABLE_BENCHMARKS "Enable microbenchmarking" ON)
+option(INSTALL_VENDORED_HEADERS "Install vendored public headers to <install_dir>/valhalla/third_party" ON)
```

Comment:
do we really need this option? i get that a person could install their own but then they have 2 new problems:

1. what version to install
2. which ones to install

id rather we dont even give the option and just install what we know will work if they want to do something special they can worry about it. or am i missing an important use-case here?

Reply:
we talked about it in review: I'll remove this option. in the current state of the repo it's pointless. eventually we'll likely get rid of any vendored libs to make peace with package managers and their requirements, but that's pending actual confirmation that vendored header-only libraries aren't ok for compile-time-only dependencies. if it's too hard, we could even incorporate those headers into our repo with proper attribution. anyways, digressing, but this was on some level part of my thought here :D